### PR TITLE
Fix for filters

### PR DESCRIPTION
**Pull request by sebworks**
Fix for filters

## Changes
- Updated  '_includes/post-macros.html' to add fix for filters.

## Test
- Visit 'http://localhost:7000/blog' and select some filter criteria. 

## Review
@jimmynotjim
@anselmbradford
@KimberlyMunoz
@kurtw

### CODE REVIEW

**Comment by anselmbradford** [comment]
:+1:

File: _includes/post-macros.html [hunk 1]
```diff
@@ -291,8 +291,8 @@ <h1 class="summary_heading">{{ post.title|safe }}</h1>
     'active_filters_total': active_filters_total
 } -%}
 
-{%- import 'macros/util/is-mobile.html' as is_mobile -%}
```
Comment (rosskarchner) [inline _includes/post-macros.html]: I apologize for being late to the party here-- but this stuff makes me a little nervous: template code fires on the server, which means the output will be cached by Akamai and served to potentially many devices.

There might be solutions here ([perhaps, instructing Akamai to honor Vary: headers, and sending Vary:user-agent](https://www.fastly.com/blog/best-practices-for-using-the-vary-header/)), so don't panic yet.

Comment (sebworks) [inline _includes/post-macros.html]: @rosskarchner, we are going to be removing this code.

Comment (rosskarchner) [inline _includes/post-macros.html]: ha, good point!

Comment (anselmbradford) [inline _includes/post-macros.html]: Removed in https://github.com/cfpb/cfgov-refresh/pull/738

**Pull request merged**